plugins/blur/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(SOURCES
     blur.cpp)
 
 if (APPLE)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
```

Comment:
I believe the issue was, you've used tabs instead of spaces. So please add back the indentation, but be sure to use spaces.